### ISSUE TITLE
Explain what deactivating your account does

### ISSUE THREAD

**Issue by turt2live** [vector-im/riot-web#6568]
> TravisR there should be a message under 'deactivate my account' explaining your username cannot be used afterward to create another account

https://matrix.to/#/!DgvjtOljKujDBrxyHk:matrix.org/$15241752872424285gXsKm:matrix.org

**Comment by uhoreg** [comment]
Doesn't "You will not be able to re-register the same user ID." already say tihs?

**Comment by turt2live** [comment]
Many people don't read the dialog, and it doesn't really hurt to have more warnings before the dialog is confirmed.

**Comment by lampholder** [comment]
Agreed - nobody reads dialogues, so if the crucial information is in there then there's a strong chance people are missing it.

**Comment by aaronraimist** [comment]
Duplicate of #5148?
